website/project/views/contributor.py

```diff
@@ -733,6 +733,9 @@ def claim_user_registered(auth, node, **kwargs):
     if should_claim:
         node.replace_contributor(old=unreg_user, new=current_user)
         node.save()
+        if isinstance(node, Registration):
+            project = node.registered_from
+            project.replace_contributor(old=unreg_user, new=current_user)
```

Comment:
What happens if the user is removed from the project before they have a chance to claim on the registration?